### DISCUSSION TITLE
Metaburner trial

### DISCUSSION THREAD

**Discussion by ivandepaz-editor** [latenitefilms/Metaburner#68] [Q&A]
Hi Chris! I'm trying to use Metaburner's trial from FxFactory but I only can get the Project TC, Frames and Seconds. I'm on FCP 10.7.1. I have selected the Extended Metadata view and all controls appear as disabled with the red ball emoji (there are not fields with "Text Layer 1", "Text Layer 2", etc...). I'm still in Ventura 13.6.2. Is this happening because of the trial or maybe because of my macOS Ventura? I don't see any watermark. Would it have a different behavior after buying it in FxFactory? I would be happy to buy it if I could use it on my current project.
![Captura de pantalla 2024-03-12 a las 1 33 29](https://github.com/latenitefilms/Metaburner/assets/57800753/1dc351f2-798f-4811-b6cf-2e11a1a08509)

**Comment by latenitefilms** [comment]
Metaburner is still very much a work-in-progress - hence the beta label in FxFactory.

We're currently doing some big under-the-hood changes, which we haven't pushed to FxFactory yet.

If you're interested in testing, here's the latest build we shared with some other testers. You'll need to manually install the Motion Template. It's still not perfect, but may be enough for what you need:

[Metaburner - 1.1.0-beta.2 (36).zip](https://github.com/latenitefilms/Metaburner/files/14566952/Metaburner.-.1.1.0-beta.2.36.zip)

Any questions let me know!

**Reply by ivandepaz-editor** [reply]
Thank you ✨

**Comment by ivandepaz-editor** [comment]
Still doesn't show me anything else more than Project TC-Frames-Seconds. I suppose I will have to update to Sonoma.
![Captura de pantalla 2024-03-12 a las 12 11 57](https://github.com/latenitefilms/Metaburner/assets/57800753/01a2ad5a-7a63-403b-9127-bffb5db4575b)

**Reply by latenitefilms** [reply]
That looks like it's still running the old version (based on the version number in the screenshot). You may have to uninstall the FxFactory version.

**Comment by ivandepaz-editor** [comment]
Thanks! I had an old Metaburner PRO in the apps folder that maybe was making the problem. After uninstalling from FxFactory and installing your last Beta it shows again as installed in FxFactory but with the 2.36 working in FCP and with the watermark that I didn't have before. 🔥 Amazing work. THANKS ✨

**Comment by ivandepaz-editor** [comment]
Hi Chris. I have just few questions (nothing urgent):
- Is it possible at the moment to show the Source File Name inside of a synced clip or the Source File Name of the active angle inside of a Multicam clip?  I've been trying "Video - Source File Name" but it doesn't show anything.
- Is it possible to show the Speed Values? Maybe with LUA? (I have no idea how to code).
- Would it be possible to show any metadata column, even a custom one? I've created for some projects 2 columns with the CDL values from the DIT and it would be great to burn them.

Thanks for you work and your attention.
-iván de paz-

**Reply by latenitefilms** [reply]
> Is it possible at the moment to show the Source File Name inside of a synced clip or the Source File Name of the active angle inside of a Multicam clip? I've been trying "Video - Source File Name" but it doesn't show anything.

This should be working in most cases already. Are you definitely using the Motion Template that was in the above download?

Feel free to share a FCPXML with anything that's not working.

> Is it possible to show the Speed Values?

It should show the speed values in the timecode:

<img width="606" alt="image" src="https://github.com/latenitefilms/Metaburner/assets/22286696/03f6058e-e07e-490c-a165-28c0dab48c6e">

> Would it be possible to show any metadata column, even a custom one?

This should already be possible - see: https://metaburner.pro/how-to-use/#custom-metadata-field

**Comment by ivandepaz-editor** [comment]
I think I'm using the right version:
![Captura de pantalla 2024-03-14 a las 11 29 56](https://github.com/latenitefilms/Metaburner/assets/57800753/ed6aeccf-111d-4c7a-ad20-211497738d71)

I've just realized that I had a speed value showing in one clip. There are other clips that don't show the Source TC so the speed value neither.

This is the timeline where I'm testing Metaburner:
https://bit.ly/4cc8mhl

I have the following problems:
- I can see Video Source Names but no Video Source Filenames
- The Video Source TC for some clips is not showing [red markers]
- The Video Source TC for some clips is wrong [chapter markers]
- The Video Source TC for some clips is displaying repeated frames (00-01-01-03-04-05-05-07...) [standard markers]
- I can't make the Custom Metadata work for "com.kopto.shotnotesx.custom.asc_sat" and "com.kopto.shotnotesx.custom.asc_sop"
![image](https://github.com/latenitefilms/Metaburner/assets/57800753/b9776690-4aa6-4a6a-8e6c-85e4605ca14c)
This Custom Metadata Fields were created by Shot Notes X.

I'm using FCP 10.7.1 and Ventura 13.6.2

By the way, at the moment is quite easy to remove the watermark with a draw mask. Other watermarked titles work also as adjustment layers so the masks and crops affect to the video underneath making more difficult to use them for free.

**Reply by latenitefilms** [reply]
Legend, thank you - this is HUGELY helpful! 

Thanks for taking the time to test things out and provide such useful feedback!

I can reproduce these issues you're seeing.

> - I can see Video Source Names but no Video Source Filenames
> - The Video Source TC for some clips is not showing [red markers]
> - The Video Source TC for some clips is wrong [chapter markers]

Interestingly, for the first problem clip (`B001_B038_0219LY_001`), the problem seems to be the connected storyline - for whatever reason it seems to be corrupting the metadata. Once I delete the connected storyline, it seems to work - so there's a logic bug there somewhere (I think we're probably not passing the lane information from the connected group).

> - The Video Source TC for some clips is displaying repeated frames (00-01-01-03-04-05-05-07...) [standard markers]

I currently have no idea why this is happening. There must be a maths bug somewhere - although it's strange it's not happening on every clip.

> I can't make the Custom Metadata work for "com.kopto.shotnotesx.custom.asc_sat" and "com.kopto.shotnotesx.custom.asc_sop"

This is something I hadn't considered - we're currently only reading the metadata from the clip on the timeline, but in this case, the metadata is stored in the parent's resources. #69

> By the way, at the moment is quite easy to remove the watermark with a draw mask. Other watermarked titles work also as adjustment layers so the masks and crops affect to the video underneath making more difficult to use them for free.

Good point! Priority for now is to just get it all working reliably. Hopefully if it's useful/reliable, people will buy it when needed.
